common/coin_support/coin_utils.h

```diff
@@ -437,4 +437,22 @@ bool verify_receive_derivation_path(const uint32_t *path, uint8_t depth);
  */
 uint16_t get_account_name(const uint32_t *path, uint16_t account_type, char *account_name, uint8_t out_len);
 
+/**
+ * @brief Generates an user readable derivation path from a uint32_t path array.
+ *        will only write the output until the out_len is reached, i.e. the result will be
+ *        truncated if the output exceeds the out_len. Won't update the output if pointer is NULL
```

Comment:
i feel now, its not good to truncate the derivation path. instead return 0.

Reply:
this could go unnoticed

![image](https://user-images.githubusercontent.com/95833994/212927526-504d6b55-5669-44ab-a33d-730fde95d082.png)


![image](https://user-images.githubusercontent.com/95833994/212927450-89fc45de-fb58-4342-b178-57e705f69de9.png)